website/docs/compute/using-api.md

```diff
@@ -0,0 +1,341 @@
+**Result:**
+
+```json
+{
+"functions": [
+    {
+        "name": "foo",
+        "namespace": "...",
+        "lang": "...",
+        "version": "...",
+        "draftVersion": "...",
+        "apiVersion": "...",
+        "fqfn": "...",
+        "fqfnURI": "...",
+    }
+]
+}
+```
+
+#### `suborbital.admin.getFunctionResults`
+
+**Description:** Returns the most recent results (up to 5) produced by the execution of the given function.  
+**Args:** An object containing `environment`, `userId`, `namespace`, `fnName`, `version`.  
+**Result:**
+
+```json
+{
+"results": [
+    {
+        "uuid": "...",
+        "timestamp": "...",
+        "response": "..."
+    }
+]
+}
+```
+
+#### `suborbital.admin.getFunctionErrors`
+
+**Description:** Returns the most recent errors (up to 5) produced by the execution of the given function.  
+**Args:** An object containing `environment`, `userId`, `namespace`, `fnName`, `version`.  
+**Result:**
+
+```json
+{
+"errors": [
+    {
+        "uuid": "...",
+        "timestamp": "...",
+        "code": 400,
+        "message": "..."
+    }
+]
+}
+```
+
+### Exec
+
+#### `suborbital.exec.run`
+
+**Description:** Executes the given function, with the provided body, params and state loaded into the function at runtime.  
+**Args:** An object containing `environment`,`userId`,`namespace`, `fnName`,`version`.  
+**Result:** The result of the executed function.
+
+### Builder
+
+#### `suborbital.builder.build`
+
+**Description:** Builds the provided code using the specified language toolchain.  
+**Args:** An object containing `language`, `environment`, `userId`, `namespace`, `fnName`, `token`.  
+**Result:** A string containing the logs for the build.  
+
+#### `suborbital.builder.getDraft`
+
+**Description:** Gets the draft for the specified runnable.  
+**Args:** An object containing `environment`, `userId`, `namespace`, `fnName`, `token`.  
+**Result:** A specified runnable.  
+
+#### `suborbital.builder.deployDraft`
+
+**Description:** Deploys the specified runnable.  
+**Args:** An object containing `environment`, `userId`, `namespace`, `fnName`, `token`.  
+**Result:** A string containing the version.  
+
+#### `suborbital.builder.testDraft`
+
+**Description:** Tests the draft for the specified runnable.  
+**Args:** An object containing `environment`, `userId`, `namespace`, `fnName`.  
+**Result:** A string containing the result.  
+
+#### `suborbital.builder.getTemplate`
+
+**Description:** Gets the template (which controls what your users see when they create a new function) for a new function of the given language.  
+**Args:** An object containing `fnName`, `language`.  
+**Result:**
+
+```rust
+//Example template for rust language 
+import { logInfo } from "@suborbital/suborbital"
+
+export function run(input: ArrayBuffer): ArrayBuffer {
+let inStr = String.UTF8.decode(input)
+
+let out = "Hello there, " + inStr
+
+return String.UTF8.encode(out)
+}
+```
```

Comment:
Should this be here? We don't have a library for Rust, and this looks like it's an AssemblyScript runnable?

Reply:
I'll change the language to `ts` and delete the comment.